yoti-sdk-api/src/main/java/com/yoti/api/client/docs/session/retrieve/CheckResponse.java

```diff
@@ -0,0 +1,21 @@
+package com.yoti.api.client.docs.session.retrieve;
+
+import java.util.List;
+
+public interface CheckResponse {
+
+    String getId();
+
+    String getState();
+
+    List<String> getResourcesUsed();
+
+    List<? extends GeneratedMedia> getGeneratedMedia();
+
+    ReportResponse getReport();
+
+    String getCreated();
```

Comment:
Do these need to be dates in other SDKs?

Reply:
getCreatedTime at least for self-documentation?

Reply:
Yes, we'll set this to be the native Date/DateTime objects in other SDKs (in UTC), Java is just a special case

Reply:
I believe it's `getCreated()` to mirror the JSON property - probably neater to keep it like this and change to a date in the major release. It could have a comment explaining that it's a date string?

Reply:
Fixed in 9063944 @davidgrayston @emmas-yoti